Automattic/wp-calypso
# Performance Profiler: Fix tab header color in IPad

Resolves https://github.com/Automattic/dotcom-forge/issues/9117

## Proposed Changes

* Fixes the color of the CrUX tab headers on IPad

|Before | After|
|-------|------|
|![CleanShot 2024-09-12 at 17 56 26@2x](https://github.com/user-attachments/assets/b97a6632-9d6d-455c-baf7-3bbe80371ddb)| ![CleanShot 2024-09-12 at 17 55 22@2x](https://github.com/user-attachments/assets/25139728-3a3f-4823-b40c-bc23766e041f)|

## Why are these changes being made?


* To match the design

## Testing Instructions



* Using Safari with the iOS Simulator installed, open the Performance Profiler in IPad
* Check the colors of the tab headers in the CrUX section is correct

## Pre-merge Checklist
